chips/nrf5x/src/peripheral_interrupts.rs

```diff
@@ -18,7 +18,7 @@ pub const CCM_AAR: u32 = 15;
 pub const WDT: u32 = 16;
 pub const RTC1: u32 = 17;
 pub const QDEC: u32 = 18;
-pub const LPCOMP: u32 = 19;
+pub const COMP: u32 = 19;
```

Comment:
That's what I saw when I changed it. You can only use one of either lpcomp or comp at a time, so I figured this might help avoid accidentally trying to use both. But maybe some chips will use one and some will use the other so I'll add lpcomp interrupt number back in.

Reply:
What does the annotation `#[cfg(feature = "nrf52")]` do?

Reply:
That is something that we're aspiring to eventually remove. This part of the code is shared among multiple nordic chips (hence `nrf5x`), so the `cfg` works like `ifdef` in C, where code is conditionally compiled depending on the board in use. It's an open todo to restructure this a bit to remove these `cfg`s, as Tock is generally opposed to using conditional compilation.